src/content/resources/xml/fedramp_values.xml

```diff
@@ -414,6 +414,49 @@
       </allowed-values>
    </value-set>
     
+    <value-set name="automation-control-id">
+        <formal-name>Automation Control IDs</formal-name>
+        <description>The control IDs for implemented requirements that are focused on automation aspects.</description>
+        <binding pattern="implemented-requirement/@control-id"/>
+        <allowed-values allow-other="no">
+            <enum value="ac-2.1" short-label="ac-2.1">ac-2.1</enum>
+            <enum value="ac-12" short-label="ac-12">ac-12</enum>
+            <enum value="ac-18.3" short-label="ac-18.3">ac-18.3</enum>
+            <enum value="au-6.1" short-label="au-6.1">au-6.1</enum>
+            <enum value="au-3.2" short-label="au-3.2">au-3.2</enum>
+            <enum value="au-6.4" short-label="au-6.4">au-6.4</enum>
+            <enum value="au-7.1" short-label="au-7.1">au-7.1</enum>
+            <enum value="at-3.4" short-label="at-3.4">at-3.4</enum>
+            <enum value="ca-7" short-label="ca-7">ca-7</enum>
+            <enum value="cm-2.2" short-label="cm-2.2">cm-2.2</enum>
+            <enum value="cm-3.1" short-label="cm-3.1">cm-3.1</enum>
+            <enum value="cm-3.6" short-label="cm-3.6">cm-3.6</enum>
+            <enum value="cm-6.1" short-label="cm-6.1">cm-6.1</enum>
+            <enum value="cm-8.3" short-label="cm-8.3">cm-8.3</enum>
+            <enum value="cm-11" short-label="cm-11">cm-11</enum>
+            <enum value="cp-10" short-label="cp-10">cp-10</enum>
+            <enum value="ir-4.1" short-label="ir-4.1">ir-4.1</enum>
+            <enum value="ir-5.1" short-label="ir-5.1">ir-5.1</enum>
+            <enum value="ir-6.1" short-label="ir-6.1">ir-6.1</enum>
+            <enum value="pe-3" short-label="pe-3">pe-3</enum>
+            <enum value="pe-8.1" short-label="pe-8.1">pe-8.1</enum>
+            <enum value="pe-11.1" short-label="pe-11.1">pe-11.1</enum>
+            <enum value="pe-13.1" short-label="pe-13.1">pe-13.1</enum>
+            <enum value="pe-13.2" short-label="pe-13.2">pe-13.2</enum>
+            <enum value="pe-13.3" short-label="pe-13.3">pe-13.3</enum>
+            <enum value="pe-15.1" short-label="pe-15.1">pe-15.1</enum>
+            <enum value="ps-4.2" short-label="ps-4.2">ps-4.2</enum>
+            <enum value="sc-23.1" short-label="sc-23.1">sc-23.1</enum>
+            <enum value="si-2.2" short-label="si-2.2">si-2.2</enum>
+            <enum value="si-3.2" short-label="si-3.2">si-3.2</enum>
+            <enum value="si-4.2" short-label="si-4.2">si-4.2</enum>
+            <enum value="si-5.1" short-label="si-5.1">si-5.1</enum>
+            <enum value="si-7.2" short-label="si-7.2">si-7.2</enum>
+            <enum value="si-7.5" short-label="si-7.5">si-7.5</enum>
+            <enum value="si-8.2" short-label="si-8.2">si-8.2</enum>
```

Comment:
Issue #309 has 37 controls. There are only 35 here.

1. AC-2(1)
2. AC-12
3. AC-18 (03)
4. AU-6(1)
5. AU-03 (02)
6. AU-06 (04)
7. AU-07 (01)
8. AT-03 (04)
9. CA-07 (low only)
10. CM-2(2)
11. CM-3(1)
12. CM-03 (06)
13. CM-6(1)
14. CM-8(3)
15. CM-8(2)
16. CM-11 (mod & Low, not high)
17. CP-10 (low only)
18. IR-4(1)
19. IR-5(1)
20. IR-6(1)
21. PE-03
22. PE-8(1)
23. PE-11 (01)
24. PE-13(1)
25. PE-13 (2)
26. PE-13 (3)
27. PE-15(1)
28. PS-4(2)
29. SC-23 (01)
30. SI-2(2)
31. SI-03 (02)
32. SI-4(2)
33. SI-04 (20)
34. SI-5(1)
35. SI-7(2)
36. SI-7(5)
37. SI-08 (02)

Reply:
Good catch.  Those missing two are added.